src/TOML/Error.hs

```diff
@@ -24,6 +26,11 @@ data TOMLError
   | DecodeError DecodeContext DecodeError
   deriving (Show, Eq)
 
+instance Exception TOMLError where
+#if MIN_VERSION_base(4,8,0)
```

Comment:
What version of ghc does base 4.8 come with? i suspect the lowest version of ghc we test against is more recent than this

Reply:
Yes please; whatever the oldest stack.yaml we have in the repo is the oldest version of deps we support